lib/python/ensembl/brc4/runnable/process_gff3.py

```diff
@@ -355,13 +343,13 @@ def normalize_gene(self, gene, functional_annotation, fail_types):
                 functional_annotation, transcript, "transcript")
             
             # Replace qualifiers
-            old_qualifiers = transcript.qualifiers
+            old_tran_qualifiers = transcript.qualifiers
```

Comment:
Based on a quick search, the acronym for transcript is `tscp`, so I would suggest to either use that or the whole world, i.e. `old_transcript_qualifiers`.

Reply:
I changed it to the full name